catalyst/dl/runner/gan.py

```diff
@@ -1,21 +1,59 @@
-from typing import Any, Mapping  # isort:skip
-
 import torch
-
-from catalyst.dl import Runner
+from collections import OrderedDict
+from typing import Any, Mapping, Dict  # isort: skip
+from catalyst.dl.callbacks import (
+    PhaseBatchWrapperCallback,
+    PhaseManagerCallback,
+    CriterionAggregatorCallback,
+)
+from catalyst.dl import (
+    Runner,
+    Callback,
+    OptimizerCallback,
+    CriterionCallback,
+)
+from catalyst.dl.experiment import GanExperiment
+from catalyst.utils.typing import (
+    Model,
+    Device,
+    Optimizer,
+    Criterion,
+    DataLoader,
+    Dataset
+)
 
 
 class GanRunner(Runner):
+    """
+    Runner for experiments with GANs
+    """
+
+    _default_experiment = GanExperiment
+    TRAIN_PARAMS = {
```

Comment:
can they be redefined?

Reply:
They can be redefined as a default params for prepare_callbacks